packages/react-native-sdk/src/components/index.ts

```diff
@@ -9,3 +9,5 @@ export * from './CallParticipantsView';
 export * from './CallControlsView';
 export * from './CallParticipantsInfoView';
 export * from './LobbyView';
+export * from './ParticipantReactions';
```

Comment:
Discussion: do we want to export this comp. and why? 🤔 Most of our integ. will use the ParticipantView directly containing this one.

Reply:
But what if they create their own component and want to use it? I didn't find any disadvantage of exporting it.

Reply:
Let's remove it!